symfony/client/src/orangehrmTimePlugin/pages/project/ProjectSave.vue

```diff
@@ -115,7 +116,7 @@ import ProjectAdminAutocomplete from '@/orangehrmTimePlugin/components/ProjectAd
 
 const defaultProjectModel = {
   name: null,
-  customer: null,
+  customer: {id: null, label: null},
```

Comment:
don't we need to handle this in `ProjectEdit.vue`?

Reply:
yeah.